concepticondata/conceptlists/RonaTas-2011-431.tsv

```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-247	247	coffin	2011	COFFIN
+RonaTas-2011-431-248	248	age, period	1789	AGE
+RonaTas-2011-431-249	249	tomb, small hill, elevated, dry place in a marsh	1773	GRAVE
+RonaTas-2011-431-250	250	dry stalk of weed		
+RonaTas-2011-431-251	251	soot	830	SOOT
+RonaTas-2011-431-252	252	ram	1344	RAM
+RonaTas-2011-431-253	253	burn	2102	BURN
+RonaTas-2011-431-254	254	frock, sheepskin waistcoat		
+RonaTas-2011-431-255	255	blackthorn, Prunus spinosa		
+RonaTas-2011-431-256	256	loan		
+RonaTas-2011-431-257	257	the navel; various parts of agricultural and other instruments,around which something turns or rotates or which is in the centre	1838	NAVEL
+RonaTas-2011-431-258	258	young of an animal, kid, puppy, lad
+RonaTas-2011-431-259	259	pounder, beater, small mortar	2459	GRINDSTONE
+RonaTas-2011-431-260	260	breast strap		
+RonaTas-2011-431-261	261	book	963	BOOK
+RonaTas-2011-431-262	262	stumpy, a small, thick man	1246	SMALL
+RonaTas-2011-431-263	263	a kind of Hungarian sheepdog, komondor		
+RonaTas-2011-431-264	264	cloak, gown, overcoat	326	CLOAK
+RonaTas-2011-431-265	265	churn, beehive, anything with a hollow part in which something canbe fitted in, a trough, tray for channelling water, the same used as a small bridge (the latter in Translyvania)	88	BEEHIVE
+RonaTas-2011-431-266	266	ash tree, Fraxinus sp.		
+RonaTas-2011-431-267	267	mellow, powdery, crumbly, rotten, mouldy	1728	ROTTEN
+RonaTas-2011-431-268	268	pear, Pyrus communis	922	PEAR
+RonaTas-2011-431-269	269	to greet, to thank	3552	GREET
+RonaTas-2011-431-270	270	the Cuman ethnic group in Hungary		
+RonaTas-2011-431-271	271	a kind of dog, a Hungarian breed of sheepdog		
+RonaTas-2011-431-272	272	a fish living at the bottom of the river, gudgeon, Gobio fluviatilis		
+RonaTas-2011-431-273	273	laughing bird, Picus viridis		
+RonaTas-2011-431-274	274	dry summer fog, rainbow	1733	RAINBOW
+RonaTas-2011-431-275	275	the name of a group of military people in Hungary,the Cuman bodyguard to the Hungarian king		
+RonaTas-2011-431-276	276	chaff, husks, winnowings, tailings, refuse of grain	2363	CHAFF (HUSK)
+RonaTas-2011-431-277	277	cause, reason	860	CAUSE
+RonaTas-2011-431-278	278	to teach	709	TEACH
+RonaTas-2011-431-279	279	sty, cattle pen, sheepfold		
+RonaTas-2011-431-280	280	cheap	1887	CHEAP
+RonaTas-2011-431-281	281	kid (of a goat)	1972	YOUNG GOAT (KID)
+RonaTas-2011-431-282	282	sperm, seminal fluid	1505	SPERM
+RonaTas-2011-431-283	283	crumb (of bread), morsel, a small piece of bread		
+RonaTas-2011-431-284	284	Russian		
+RonaTas-2011-431-285	285	lion	1386	LION
+RonaTas-2011-431-286	286	spindle, whorl	277	SPINDLE
+RonaTas-2011-431-287	287	thief	1160	THIEF
+RonaTas-2011-431-288	288	physician	597	PHYSICIAN
+RonaTas-2011-431-289	289	cune in play in ostábla		
+RonaTas-2011-431-290	290	ox	1169	OX
+RonaTas-2011-431-291	291	to stitch, to put on a dress		
+RonaTas-2011-431-292	292	hawk, buzzard, Buteo buteo	307	HAWK
+RonaTas-2011-431-293	293	to take root, to grow stronger (plant, animal)	619	ANIMAL
```

Comment:
I'd unmap, this does not refer to `ANIMAL`

Reply:
Resolved in 70cb680e6327a35615c7b0841c50828f1bee1534